src/index.css

```diff
@@ -1,6 +1,6 @@
 :root {
   /* change this value for the background colour! */
-  --bg-color: #3535EC;
+  --bg-color: #32323f;
```

Comment:
Is this now out of date based on the background colour @nicopicchio added to the README?